File: words/Words.java
import java.util.Scanner;

public class Words {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the text:");
        String text = scanner.nextLine();
        String[] words= text.split(" ");
        int count= words.length;
        System.out.println("Your text contains " +count+ " words");

    }
}

File: words/words.py
s = input("enter your text: ")
if len(s.split()) > 1:       # If the text is multi word, it will count the number of words
    n = len(s.split())
    print("your text contains " + str(n) + " words")
elif len(s.split()) == 1:    # If the text is one word, it will count the number of characters
    a = len(s.strip())
    print("your text contains " + str(a) + " characters")